nixopsaws/resources/s3_bucket.py

```diff
@@ -197,3 +197,8 @@ def region_to_s3_location(region):
     if region == "eu-west-1": return "EU"
     elif region == "us-east-1": return "US"
     else: return region
+
+def region_name_to_region(region):
+    if region == "EU": return "eu-west-1"
+    elif region == "US": return "us-east-1"
+    else: return region
```

Comment:
I think if possible we should get rid of all these US/us-east-1, EU/eu-west-1 default stuff and always specify the region in the boto3 request since it is already required. cc @AmineChikhaoui

Reply:
@PsyanticY I can do it in this PR if you want.

Reply:
@AmineChikhaoui  should decide that ^^